scripts/build/build-hof.js

```diff
@@ -11,8 +12,11 @@ import renderApp from './renderApp'
 import write from './write-html'
 
 import rootReducer from '../../src/reducers'
-import { getInitialState } from '../../src/getInitialState'
 import { fetchAllHeroes } from '../../src/actions/hofActions'
+import {
+  fetchProjectsFromAPI,
```

Comment:
__[eslint]__
* Error - 'fetchProjectsFromAPI' is defined but never used. (no-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->